src/transformers/__init__.py

```diff
@@ -546,8 +554,7 @@
     _import_structure["models.glpn"].append("GLPNFeatureExtractor")
     _import_structure["models.imagegpt"].append("ImageGPTFeatureExtractor")
     _import_structure["models.layoutlmv2"].append("LayoutLMv2FeatureExtractor")
-    _import_structure["models.layoutlmv2"].append("LayoutLMv2Processor")
-    _import_structure["models.layoutxlm"].append("LayoutXLMProcessor")
```

Comment:
The processors shouldn't be under a `is_vision_available`, so I removed them for LayoutLMv2 and LayoutXLM.